src/components/order-service/order-service-review.html

```diff
@@ -1,27 +1,42 @@
 <div class="col-md-12 center">
-  <div class="title">Your Order is Complete! <span class="fa fa-check success-check"></span></div>
-  <div class="sub-title center">Launch this service now to access your project directly.</div>
-  <div class="launch-service">
-    <button type="button" class="btn btn-default btn-primary order-btn" ng-click="$ctrl.cancelOrder()">
-      Launch Service
-    </button>
+  <div ng-if="!$ctrl.error">
+    <div class="title">Your Order is Complete! <span class="fa fa-check success-check"></span></div>
```

Comment:
Agree.

I think we need to look at the wording used in the dialog in general. I'm not sure we're using the right terminology.